static/app/components/replays/replayContext.tsx

```diff
@@ -420,8 +420,11 @@ export function Provider({children, replay, initialTimeOffset = 0, value = {}}:
   }, []);
 
   // Only on pageload: set the initial playback timestamp
-  // Don't include `setCurrentTime` in the hook deps array because it changes
+  // Do not include `setCurrentTime` in the hook deps array because it changes
   // on each play/pause state change.
+  // Do include replayerRef.current in the hook deps. The rule warns that since
```

Comment:
Specifically tracking `ref.current` will cause the effect to run when a render is happening if that value has changed. This was an interesting note I took away from our similar convo: https://github.com/getsentry/sentry/pull/36193#discussion_r910275799